trunk/NDHMS/HYDRO_drv/module_HYDRO_drv.F

```diff
@@ -1152,9 +1153,9 @@ subroutine aggregateDomain(did)
                    endif
 
 !end 
-                    IF (RT_DOMAIN(did)%SMCRT(IXXRT,JYYRT,KRT) .GT. &
-                        RT_DOMAIN(did)%SMCMAXRT(IXXRT,JYYRT,KRT)) THEN
 
+!ADCHANGE: Change to accept very small diff since was being triggered by vals that print identical
+IF ( (RT_DOMAIN(did)%SMCRT(IXXRT,JYYRT,KRT)-RT_DOMAIN(did)%SMCMAXRT(IXXRT,JYYRT,KRT)) .GT. 0.000001 ) THEN
```

Comment:
This seems to work well (solved some issues I was having), but I wonder about the hard coded value.  

Seems like there are maybe [better practices](http://jules-lsm.github.io/coding_standards/guidelines/fp_arithmetic.html).  Also, for reference, Noah / Noah deal w/ this by defining a constant [here](https://github.com/NCAR/wrf_hydro_nwm_public/blob/master/trunk/NDHMS/Land_models/Noah/Utility_routines/module_model_constants.F).